the-badass-todo-list-part-1/input.js

```diff
@@ -2,6 +2,7 @@
 
 (function() {
   function drawInput(motd) {
+    // Quiz #2
```

Comment:
You got it. You will see this get resolved in the following examples as a data member and be initialized at the beginning of the managers.